saleor/checkout/fetch.py

```diff
@@ -222,13 +242,31 @@ def fetch_checkout_info(
 ) -> CheckoutInfo:
     """Fetch checkout as CheckoutInfo object."""
 
+    from .utils import get_app_shipping_id
+
     channel = checkout.channel
     shipping_address = checkout.shipping_address
+    shipping_channel_listings = None
+
```

Comment:
We can move `delivery_method = checkout.collection_point` here as a default and remove further `else`